cmd/storj-sim/network.go

```diff
@@ -262,10 +262,13 @@ func newNetwork(flags *Flags) (*Processes, error) {
 		consoleAuthToken := "secure_token"
 
 		var schema string
-		database := "sqlite3://" + process.Directory + "/master.db"
+		sqlDatabase := "sqlite3://" + process.Directory + "/master.db"
+		kvDatabase := "bolt://" + process.Directory + "pointers.db"
```

Comment:
You might be able to avoid this, if you add the argument dynamically when needed.

Reply:
good idea, let me try that